tests/core/test_composite.py

```diff
@@ -334,8 +334,8 @@ def test_multi_block_repr(ant, sphere, uniform, airplane):
     assert str(multi) is not None
 
 
-def test_multi_block_eq(ant, sphere, uniform, airplane, globe):
-    multi = multi_from_datasets(ant, sphere, uniform, airplane, globe)
```

Comment:
Should these tests still be performed on `globe` among the other texture-dependent tests? It's unclear whether testing `globe` here actually covered any unique cases (I'm not sure if the presence of a texture affects any of these), but I ask because this seems to be one of the few functional changes made to the tests.

Reply:
It *shouldn't* -- I'm hoping that codecov is accurately representing this too by not showing any coverage differences. The texture tracking functionalities should be fully captured by `test_copy_metadata` in `tests/core/test_dataset.py`

Additionally, I am deprecating the texture tracking in a follow up PR so if this does affect coverage, that will be deprecated in the next week

Reply:
In an earlier state of the PR there were only 2 uncovered lines and those were accounted for by a specific test (discussed in a thread). So I'm fairly sure we're not losing coverage due to this change.